test/NuGet.Core.Tests/NuGet.Credentials.Test/PluginManagerMock.cs

```diff
@@ -13,10 +13,11 @@
 using NuGet.Protocol;
 using NuGet.Protocol.Plugins;
 using NuGet.Test.Utility;
-using NuGet.Versioning;
 
 namespace NuGet.Credentials.Test
 {
+    using SemanticVersion = Versioning.SemanticVersion;
```

Comment:
Fixes compile error. I can't recall which change caused this error to start happening...